tests/generic/test_datacontainer.py

```diff
@@ -459,6 +460,54 @@ def test_subclass_preservation(self):
         )
         self.pl.pop('subclass')
 
+    def test_stub(self):
+        """Lazily loaded containers should contain only stubs and only force them when directly accessed."""
+
+        self.pl.to_hdf(self.hdf, "lazy")
+        ll = self.hdf["lazy"].to_object(lazy=True)
+        self.assertTrue(all(isinstance(v, HDFStub) for v in ll._store),
+                        "Not all values loaded as stubs!")
+
+        ll_repr = repr(ll)
```

Comment:
Codacy also complains this is unused. If I understand, you're just making sure the repr doesn't throw an error, yeah? In that case I guess there's no need to assign it to a variable. Similar down in line 475.

Reply:
It's to make sure that calling `repr` doesn't force the stub inside a lazily loaded container, I'll remove it, but I fear codacy will still complain.